src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -286,14 +286,30 @@ private async Task<ExtendLedgerReport> ProcessTransactions(ReadWriteDbContext db
                         var coreBlobs = userLedgerTransaction.NotarizedTransaction.SignedIntent.Intent.BlobsHex;
                         using var manifestInstructions = ToolkitModel.Instructions.FromString(coreInstructions, _networkConfigurationProvider.GetNetworkId());
                         using var toolkitManifest = new ToolkitModel.TransactionManifest(manifestInstructions, coreBlobs.Values.Select(x => x.ConvertFromHex()).ToArray());
-                        var extractedAddresses = ManifestAddressesExtractor.ExtractAddresses(toolkitManifest);
+
+                        var extractedAddresses = ManifestAddressesExtractor.ExtractAddresses(toolkitManifest, _networkConfigurationProvider.GetNetworkId());
 
                         foreach (var address in extractedAddresses.All())
                         {
                             referencedEntities.MarkSeenAddress(address);
                         }
 
                         manifestExtractedAddresses[stateVersion] = extractedAddresses;
+
+                        var manifestSummary = toolkitManifest.Summary(_networkConfigurationProvider.GetNetworkId());
+
+                        for (var i = 0; i < manifestSummary.classification.Length; ++i)
```

Comment:
Hint: do you know `++i` has literally no effect compared to regular `i++` in this case? :)

Reply:
i know it has no difference in c#, i simply always use ++i:P